### PR TITLE
Updated version of mobile portfolio.

### PR DESCRIPTION
**Pull request by Itswali**
This is a mobile version of Portfolio.

I added HTML file with content.
I added CSS to design the HTML content as per requiremnt.
I added .gitignore file to filter the unwanted files.
I added the linters to solve the error in HTML and CSS files.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -62,26 +64,391 @@ <h3>LET'S CONNECT</h3>
...
+
+                    <p>Back End Dev</p>
+
+                    <img src="./Images/header/Counter.png" alt="dott">
+                    <p>2015</p>
+
+
+
+                </div>
+
```
Comment (jerryowusu) [inline index.html:102]: - [ ] As stated in the project requirement above, you are supposed to create two sections at these stage of the project, one section for the `works/projects` and another section for the `about me`. 

 No need to have several separated divs. You can put each individual card container inside a div tag which will also be inside one section. Kindly put lines 78 to 300 into  `works/projects` section and lines  305 to 417 into `about me` section. 

- [ ] Don't forget to also use display:grid to align the content of the section container.

Screenshot from 2022-09-24 12-11-24

**Review by jerryowusu — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (jerryowusu) [inline index.html]: - [ ] As stated in the project requirement above, you are supposed to create two sections at these stage of the project, one section for the `works/projects` and another section for the `about me`. 

 No need to have several separated divs. You can put each individual card container inside a div tag which will also be inside one section. Kindly put lines 78 to 300 into  `works/projects` section and lines  305 to 417 into `about me` section. 

- [ ] Don't forget to also use display:grid to align the content of the section container.

Screenshot from 2022-09-24 12-11-24

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @Itswali 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

soclose


 ## Highlights
:heavy_check_mark:  No Linter error
:heavy_check_mark:  Correct use of GitHub flow

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Olamarx — APPROVED** [review]
Hi @Itswali, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**